integration/load.php

```diff
@@ -146,13 +146,27 @@ function ( $transformer, $data, $object_class ) {
 	 * @see https://wordpress.org/plugins/surge/
 	 */
 	Surge::init();
+
+	/**
+	 * Load the LiteSpeed Cache integration.
+	 *
+	 * The check for whether LiteSpeed Cache is loaded and initialized happens inside Litespeed_Cache::init().
+	 *
+	 * @see https://wordpress.org/plugins/litespeed-cache/
+	 */
+	Litespeed_Cache::init();
 }
 \add_action( 'plugins_loaded', __NAMESPACE__ . '\plugin_init' );
 
 // Register activation and deactivation hooks for Surge integration.
 \register_activation_hook( ACTIVITYPUB_PLUGIN_FILE, array( __NAMESPACE__ . '\Surge', 'add_cache_config' ) );
 \register_deactivation_hook( ACTIVITYPUB_PLUGIN_FILE, array( __NAMESPACE__ . '\Surge', 'remove_cache_config' ) );
 
+// Register activation and deactivation hooks for LiteSpeed Cache integration.
+\register_activation_hook( ACTIVITYPUB_PLUGIN_FILE, array( __NAMESPACE__ . '\LiteSpeed_Cache', 'add_htaccess_rules' ) );
+\register_deactivation_hook( ACTIVITYPUB_PLUGIN_FILE, array( __NAMESPACE__ . '\LiteSpeed_Cache', 'remove_htaccess_rules' ) );
```

Comment:
With this deactivation hook we should be able to remove the changes in `includes/class-activitypub.php`.